### PR TITLE
Prevent Silent Deletion of Instances in Annotation_Update

### PR DESCRIPTION
**Pull request by PJEstrada**
In some cases the frontend can have unexpected bugs which makes the instance_list to not be sent completely to the backend.

Old versions assume that if instance was not present, it should be deleted, which caused unexpected silent deletion in some cases.

Now we double check that all the instances are available on the request payload before executing any logic in Annotation_Update

### CODE REVIEW

**Comment by PJEstrada** [comment]
Now when this cases happen we will see a message similar to this:
image


Finishing e2e tests and we should be good to go!

**Comment by PJEstrada** [comment]
E2e tests and unit tests are all ready, waiting for them to pass and for any feedback before merging :)

File: shared/annotation.py
Comment (anthony-sarkis) [inline shared/annotation.py:380]: suggest removing print before merging unless we want to log it fully

**Review by anthony-sarkis — APPROVED** [review]

File: shared/annotation.py
Comment (anthony-sarkis) [inline shared/annotation.py:374]: Good catch :)

Comment (PJEstrada) [reply shared/annotation.py:374]: Yes Thanks for noticing that one Anthony :)

**Pull request merged**
